package/MDAnalysis/analysis/density.py

```diff
@@ -689,6 +691,23 @@ def density_from_Universe(universe, delta=1.0, atomselection='name OH2',
       (It should be noted that the `padding` keyword is not used when a user
       defined grid is assigned).
 
+    As detailed above, the :class:`Density` object returned contains a
+    physical density in units of Angstroem^{-3}. If you are interested in
+    recovering the underlying probability density, simply divide by the sum::
+
+      physical_density = density_from_Universe(universe, delta=1.0,
+                                               atomselection='name OW')
+
+      probability_density = physical_density / physical_density.grid.sum()
+
+    Similarly, if you would like to recover a grid containing a histogram of
+    atom counts, simply multiply by the volume::
+
+      atom_count_histogram = physical_density * (physical_density.delta[0] ** 3)
```

Comment:
Is `delta` the same in all directions?

Reply:
Good shout :)

So the code calls `fixedwidth_bins`, which allegedly will always return a set of bins that obey the input `delta` value. Since a single `delta` currently can only be defined by users, delta is the same in all directions. However, maybe it might be safer to not make this assumption in the event that future versions of the density code allow for non-uniform bin widths?

Reply:
@RMeli this should be fixed now